packages/forms-web-app/src/views/layouts/vertical-tabs.njk

```diff
@@ -23,8 +23,9 @@
     </li>
     <li class="side-menu govuk-body-s {% if 'representations' == activeProjectLink %} active {% endif %}">
       {% if 'representations' != activeProjectLink %}
-        <a href="/projects/{{ caseRef }}/representations">Registration comments</a>
-      {% else %} Registration comments {% endif %}
+        <a href="/projects/{{ caseRef }}/representations">Registration comments </br> (Relevant representations)
+        </a>
+      {% else %} Registration comments </br> (Relevant representations) {% endif %}
```

Comment:
Same, here is the </br> intentional?

Reply:
Yes, I did it intentionally, only because in the prototype it is on separate lines (I like the way it looks better) -- what do you think?